rebasehelper/utils.py

```diff
@@ -285,61 +287,54 @@ def format_size(s):
         sys.stdout.flush()
 
     @staticmethod
-    def download_file(url, destination_path, timeout=10, blocksize=8192):
+    def download_file(url, destination_path, blocksize=8192):
         """
         Method for downloading file from HTTP, HTTPS and FTP URL.
 
         :param url: URL from which to download the file
         :param destination_path: path where to store downloaded file
-        :param timeout: timeout in seconds for blocking actions like connecting, etc.
         :param blocksize: size in Bytes of blocks used for downloading the file and reporting progress
         :return: None
         """
+        session = requests_ftp.ftp.FTPSession()
+        r = session.get(url)
+        if not 200 <= r.status_code < 300:
+            raise DownloadError(r.url, r.status_code, r.reason, r.headers, None)
```

Comment:
`DownloadError` should contain reason string only, see https://github.com/rebase-helper/rebase-helper/blob/master/rebasehelper/specfile.py#L163-L165.